### PR TITLE
Test windows sensor

### LINKED ISSUE #3: Agents are not automatically registered
Hi @sonofagl1tch,

After deploying the detonation lab, I had to register the agents manually. I think the registration scripts should be improved to use [authd](https://documentation.wazuh.com/current/user-manual/registering/use-registration-service.html).

Currently defined tasks:
* [x] Add port 1515 to wazuh manager's security group (the wazuh agent doesnt need to have any opened port).
* [x] Enable `authd` in the wazuh manager.
* [x] Modify registration script for linux agents.
* [x] Modify registration script for windows agents using the MSI (https://documentation.wazuh.com/current/installation-guide/installing-wazuh-agent/wazuh_agent_windows.html#using-the-command-line).

Best regards,
Marta

### PR DESCRIPTION
**Pull request by sonofagl1tch**
closes #3 

*Description of changes:*
added command to run powershell script after download
changed template userdata commands for basicWindowsTarget resource

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsDetonationLab.template [hunk 1]
```diff
@@ -3049,6 +3021,48 @@
           "Ref": "wazuhUser"
         }
       }
+    },
+    "KinesisToLambda": {
```
Comment (mgmacias95) [inline awsDetonationLab.template]: Is this change really necessary in this PR?

Comment (sonofagl1tch) [reply awsDetonationLab.template]: I am pretty sure this change is just the lines being moved due to the actual changes

Comment (mgmacias95) [reply awsDetonationLab.template]: You're right. I have checked these changes are present in master as well 😄 .

**Comment by sonofagl1tch** [comment]
branch took too many changes that were not relevant. closing pull request to delete the branch.
